packages/driver/cypress/e2e/commands/actions/click.cy.js

```diff
@@ -546,6 +546,27 @@ describe('src/cy/commands/actions/click', () => {
       cy.getAll('span2', 'focus click mousedown').each(shouldNotBeCalled)
     })
 
+    // https://github.com/cypress-io/cypress/issues/28788
+    it('no click when element is disabled', () => {
+      const btn = cy.$$('button:first')
+      const span = $('<span>foooo</span>')
+
+      attachFocusListeners({ btn, span })
+      attachMouseClickListeners({ btn, span })
+      attachMouseHoverListeners({ btn, span })
+
+      btn.html('')
+      btn.attr('disabled', true)
+      btn.append(span)
+
+      cy.get('button:first span').click()
+
+      cy.getAll('btn', 'mouseenter mousedown mouseup').each(shouldBeCalled)
```

Comment:
OK looks like chrome fires the mouse enter event on the parent element as well, but not in electron. We should be able to do something like this, or remove the assertion
```suggestion
      if (Cypress.browser.name === 'chrome') {
          cy.getAll('btn', 'mouseenter mousedown mouseup').each(shouldBeCalled)
      }
```